src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -154,7 +154,7 @@ HandlerResolver::GetApplicationFactory(const IHttpApplication& pApplication, std
 
             errorContext.statusCode = 500i16;
             errorContext.subStatusCode = 35i16;
-            errorContext.generalErrorType = "ANCM Multiple In-Process Applications in same Process";
+            errorContext.generalErrorType = "ASP.NET Core Module Multiple In-Process Applications in same Process";
```

Comment:
```suggestion
            errorContext.generalErrorType = "ASP.NET Core does not support multiple apps in the same app pool";
```

(yes, I know out-of-proc does but that’s a nuance for docs IMO). Also "App Pool" is, I believe, the more public-facing IIS term here.